Cargo.toml

```diff
@@ -126,9 +126,9 @@ resvg = { version= "0.34.0", default-features = false, features = ["text"] }
 fontdb = { version = "0.14.1", default-features = false }
 send_wrapper = { version = "0.6.0" }
 cbindgen = { version = "0.26", default-features = false }
-toml_edit = "0.20"
+toml_edit = { version = "0.20" }
 css-color-parser2 = { version = "1.0.1" }
-itertools = "0.11"
+itertools = { version = "0.11" }
```

Comment:
It seems fragile that this needs to be like that without a test on the CI that checks it.
I wouldn't want to have the surprice that the node package upload doesn't work when we want to do the release because we added a dependency and we didn't use the right format for the xtask job to run.

I think it'd be nice to run the xtask job as dry run in the CI

Reply:
Yes that’s what I mentioned in the chat. I’ll work on a ci job that creates a package, installs it, and builds it.